plugin/adsk/plugin/ProxyShape.h

```diff
@@ -15,11 +15,12 @@
 //
 #pragma once
 
-#include "base/api.h"
+#include <mayaUsdPlugin/base/api.h>
```

Comment:
This is a private header, and should be "base/api.h" (and moved to end, regardless)